.github/workflows/tests.yml

```diff
@@ -240,30 +262,18 @@ jobs:
           cache-dependency-glob: |
             pyproject.toml
 
-      - name: Build package
-        run: |
-          uv pip install build
-          make dist
-
-      - name: Install wheel; test CLI + models with assets
+      - name: Build from source
         run: |
-          uv venv .venv-whl
-          PYTHON_BIN=.venv-whl/bin/python
-          uv pip install --python=$PYTHON_BIN zamba@$(find dist -name 'zamba*.whl') --force-reinstall
-          $PYTHON_BIN -m zamba --help
-          $PYTHON_BIN -c "from zamba.data.video import MegadetectorLiteYoloX; MegadetectorLiteYoloX()"
+          uv pip install .
 
-      - name: Install source; test CLI + models with assets
+      - name: Test import and model loading
         run: |
-          uv venv .venv-sdist
-          PYTHON_BIN=.venv-sdist/bin/python
-          uv pip install --python=$PYTHON_BIN zamba@$(find dist -name 'zamba*.tar.gz') --force-reinstall
```

Comment:
I think we were installing the `.tar.gz` to most closely mimic what users will be installing. do you think that's unnecessary?

Reply:
It _should_ be generally equivalent. In both cases, the installer (pip or uv) will install the declared build backend (in our case, `flit_core`), which will build a wheel from the source and then install the wheel. 

Having people install from sdist (the `.tar.gz` distribution) just inserts an unnecessary step of having to generate the sdist file, which is all of the same files the wheel build step is going to use but just in a different format. 

In fact, we should probably attach the wheel to the releases and have people install from the wheel file instead. It will save a step and be slightly faster for users.